### PR TITLE
CTONET-780 - Test pod priviliges

### PR DESCRIPTION
**Pull request by ydayagi**
We read pod's service account and use it to understand the roles it
binds to

https://issues.redhat.com/browse/CTONET-780

### CODE REVIEW

File: pkg/tnf/handlers/clusterrolebinding/clusterrolebinding_test.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/clusterrolebinding/clusterrolebinding_test.go:1]: 2021

File: pkg/tnf/handlers/clusterrolebinding/doc.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/clusterrolebinding/doc.go:1]: 2021

File: pkg/tnf/handlers/rolebinding/doc.go [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/rolebinding/doc.go:1]: 2021

File: pkg/tnf/handlers/rolebinding/rolebinding.go [hunk 4]
```diff
@@ -0,0 +1,124 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/rolebinding/rolebinding.go:1]: 2021

File: pkg/tnf/handlers/rolebinding/rolebinding_test.go [hunk 5]
```diff
@@ -0,0 +1,113 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/rolebinding/rolebinding_test.go:1]: 2021

File: pkg/tnf/handlers/serviceaccount/doc.go [hunk 6]
```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/serviceaccount/doc.go:1]: 2021

File: pkg/tnf/handlers/serviceaccount/serviceaccount.go [hunk 7]
```diff
@@ -0,0 +1,97 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/serviceaccount/serviceaccount.go:1]: 2021

File: pkg/tnf/handlers/serviceaccount/serviceaccount_test.go [hunk 8]
```diff
@@ -0,0 +1,288 @@
+// Copyright (C) 2020 Red Hat, Inc.
```
Comment (ryandgoulding) [inline pkg/tnf/handlers/serviceaccount/serviceaccount_test.go:1]: 2021

File: pkg/tnf/identifier/identifiers.go [hunk 9]
```diff
@@ -199,3 +202,21 @@ var VersionIdentifier = Identifier{
...
+var RoleBindingIdentifier = Identifier{
+	URL:             roleBindingIdentifierURL,
+	SemanticVersion: versionOne,
+}
+
+// ClusterRoleBindingIdentifier is the Identifier used to represent the generic clusterRoleBinding test.
+var ClusterRoleBindingIdentifier = Identifier{
+	URL:             clusterRoleBindingIdentifierURL,
+	SemanticVersion: versionOne,
+}
```
Comment (ryandgoulding) [inline pkg/tnf/identifier/identifiers.go:222]: Please add appropriate entries to the Catalog map here:
https://github.com/test-network-function/test-network-function/blob/main/pkg/tnf/identifier/identifiers.go#L70

**Comment by ryandgoulding** [comment]
Please fix the lint errors exposed by running `make lint`.

**Review by ryandgoulding — CHANGES_REQUESTED** [review]

**Comment by ryandgoulding** [comment]
Looks great in general.  I have a few small suggestions inlined aside from the `lint` issue above.  Great work!

**Review by ryandgoulding — APPROVED** [review]

**Review by crwr45 — APPROVED** [review]

**Pull request merged**